app/src/main/java/org/wikipedia/page/PageFragment.kt

```diff
@@ -807,14 +809,17 @@ class PageFragment : Fragment(), BackPressedHandler, CommunicationBridge.Communi
         bridge.addListener("header_item") { _, messagePayload ->
             messagePayload?.let { payload ->
                 when (payload["itemType"]?.jsonPrimitive?.content) {
-                    "talkPage" -> sidePanelHandler.showTalkTopics()
+                    "talkPage" -> {
+                        TalkFunnel(model.title!!, InvokeSource.PAGE_TALK_BUBBLE).pageTalkBubbleClick()
+                        sidePanelHandler.showTalkTopics()
+                    }
                 }
             }
         }
         bridge.addListener("footer_item") { _, messagePayload ->
             messagePayload?.let { payload ->
                 when (payload["itemType"]?.jsonPrimitive?.content) {
-                    "talkPage" -> model.title?.run { startTalkTopicActivity(this) }
+                    "talkPage" -> model.title?.run { startTalkTopicActivity(this, PAGE_SIDE_PANEL) }
```

Comment:
This is actually from the footer item on the article page instead of the side panel. Would you like to confirm with Robin or Jazmin about this? (e.g. does the footer "Talk page" button open the side panel or go to the talk page directly?).